index.ts

```diff
@@ -44,15 +68,18 @@ async function main() {
                 // if issue exists in devpool
                 const devpoolIssue = getIssueByLabel(devpoolIssues, `id: ${projectIssue.node_id}`);
                 if (devpoolIssue) {
+                  const additionalLabelsToAdd = projectIssue?.assignee?.login
+                    ? ["Unavailable"]
+                    : [];
                     // if title or state or pricing was updated then update a devpool issue
-                    if (devpoolIssue.title !== projectIssue.title || devpoolIssue.state !== projectIssue.state || getIssuePriceLabel(devpoolIssue) !== getIssuePriceLabel(projectIssue)) {
+                    if (devpoolIssue.title !== projectIssue.title || devpoolIssue.state !== projectIssue.state || devpoolIssue.assignee?.login !== projectIssue?.assignee?.login || projectIssue?.assignee?.login === undefined || getIssuePriceLabel(devpoolIssue) !== getIssuePriceLabel(projectIssue)) {
```

Comment:
`projectIssue?.assignee?.login === undefined` means that an issue will be updated **every time** if there is no assignee. Github API calls are expensive in terms of rate limits. Basically we should this condition here `if ("Unavailable" label exists and there is no partner issue's assignee OR "Unavailable" label does not exist and there is partner issue's assignee) then update an issue`

Reply:
Fixed